test/tokens/_api.scss

```diff
@@ -131,8 +131,23 @@
           'ERROR [lookup-alias] Alias "that->baz" not found in map'
         )
       );
+
+      config.$handle-missing-keys: null;
+    }
+
+    @include it('Allows CSS values containing hashes') {
+      $gradients: (
+        'accent': linear-gradient(#f88812 10%, #9f29fa 80%),
```

Comment:
Interesting side-note: Sass currently treats unknown/css functions as strings, which is part of the issue in this specific case. We're talking about what it might mean to add a CSS function data type…

Reply:
Very interesting indeed - thanks for the insight.

Perhaps related: since unquoted strings have to be valid CSS they behave quite differently from quoted strings. I came up against the fact that there is no `string.is-quoted()` fn in sass:string, which seems like an oversight; but then I decided I didn't need it.

Reply:
@johncrim Can you file an issue on the Sass repo? I think it would be good to document some of the use-cases for it. Now that we have modules, we're looking into what functions are missing from them.

Reply:
oh, I guess if you didn't need it, maybe there's nothing there?

Reply:
Hi @mirisuzanne  - there's a [long-standing issue asking for `string.is-quoted()`](https://github.com/sass/sass/issues/1291).

I'll add a comment to that issue.